<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1117</width>
    <height>689</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTabWidget" name="tabWidget">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(66, 66, 66);</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Isolator</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QGroupBox" name="groupbox_4">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(189, 189, 189);</string>
             </property>
             <property name="title">
              <string/>
             </property>
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="0" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QPushButton" name="pause_button">
                  <property name="minimumSize">
                   <size>
                    <width>50</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>pause 1.png</normaloff>pause 1.png</iconset>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="play_button">
                  <property name="minimumSize">
                   <size>
                    <width>50</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>play.png</normaloff>play.png</iconset>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSlider" name="volume_slider">
                  <property name="minimumSize">
                   <size>
                    <width>400</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="volume_number">
                  <property name="minimumSize">
                   <size>
                    <width>150</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;</string>
                  </property>
                  <property name="text">
                   <string>Volume</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QGroupBox" name="groupbox">
               <property name="maximumSize">
                <size>
                 <width>640</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(107, 107, 107);</string>
               </property>
               <property name="title">
                <string/>
               </property>
               <layout class="QGridLayout" name="gridLayout_6">
                <item row="0" column="0">
                 <layout class="QVBoxLayout" name="verticalLayout_5">
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_4">
                    <item>
                     <layout class="QVBoxLayout" name="verticalLayout">
                      <item>
                       <widget class="QLabel" name="instrument3_label">
                        <property name="maximumSize">
                         <size>
                          <width>80</width>
                          <height>80</height>
                         </size>
                        </property>
                        <property name="styleSheet">
                         <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Eras Demi ITC&quot;;</string>
                        </property>
                        <property name="text">
                         <string/>
                        </property>
                        <property name="pixmap">
                         <pixmap>C:/Users/User/Downloads/drumsmelawen.png</pixmap>
                        </property>
                        <property name="scaledContents">
                         <bool>true</bool>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <spacer name="verticalSpacer_2">
                        <property name="orientation">
                         <enum>Qt::Vertical</enum>
                        </property>
                        <property name="sizeType">
                         <enum>QSizePolicy::Minimum</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                         <size>
                          <width>20</width>
                          <height>20</height>
                         </size>
                        </property>
                       </spacer>
                      </item>
                      <item>
                       <widget class="QSlider" name="instrument1_slider">
                        <property name="orientation">
                         <enum>Qt::Horizontal</enum>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                    <item>
                     <layout class="QVBoxLayout" name="verticalLayout_2">
                      <item>
                       <widget class="QLabel" name="instrument1_label">
                        <property name="minimumSize">
                         <size>
                          <width>0</width>
                          <height>0</height>
                         </size>
                        </property>
                        <property name="maximumSize">
                         <size>
                          <width>80</width>
                          <height>80</height>
                         </size>
                        </property>
                        <property name="styleSheet">
                         <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Eras Demi ITC&quot;;</string>
                        </property>
                        <property name="text">
                         <string/>
                        </property>
                        <property name="pixmap">
                         <pixmap>pianofinal.jpeg</pixmap>
                        </property>
                        <property name="scaledContents">
                         <bool>true</bool>
                        </property>
                        <property name="wordWrap">
                         <bool>false</bool>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <spacer name="verticalSpacer">
                        <property name="orientation">
                         <enum>Qt::Vertical</enum>
                        </property>
                        <property name="sizeType">
                         <enum>QSizePolicy::Minimum</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                         <size>
                          <width>20</width>
                          <height>20</height>
                         </size>
                        </property>
                       </spacer>
                      </item>
                      <item>
                       <widget class="QSlider" name="instrument2_slider">
                        <property name="maximumSize">
                         <size>
                          <width>1677215</width>
                          <height>16777215</height>
                         </size>
                        </property>
                        <property name="orientation">
                         <enum>Qt::Horizontal</enum>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                    <item>
                     <layout class="QVBoxLayout" name="verticalLayout_3">
                      <item>
                       <widget class="QLabel" name="instrument2_label">
                        <property name="maximumSize">
                         <size>
                          <width>80</width>
                          <height>80</height>
                         </size>
                        </property>
                        <property name="styleSheet">
                         <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Eras Demi ITC&quot;;</string>
                        </property>
                        <property name="text">
                         <string/>
                        </property>
                        <property name="pixmap">
                         <pixmap>saxofinal.png</pixmap>
                        </property>
                        <property name="scaledContents">
                         <bool>true</bool>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <spacer name="verticalSpacer_3">
                        <property name="orientation">
                         <enum>Qt::Vertical</enum>
                        </property>
                        <property name="sizeType">
                         <enum>QSizePolicy::Minimum</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                         <size>
                          <width>20</width>
                          <height>20</height>
                         </size>
                        </property>
                       </spacer>
                      </item>
                      <item>
                       <widget class="QSlider" name="instrument3_slider">
                        <property name="orientation">
                         <enum>Qt::Horizontal</enum>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <widget class="QPushButton" name="equalize_button">
                    <property name="styleSheet">
                     <string notr="true">background-color: rgb(189, 189, 189);</string>
                    </property>
                    <property name="text">
                     <string>Equalize</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QSplitter" name="splitter_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <widget class="PlotWidget" name="mainsignal_widget">
                <property name="minimumSize">
                 <size>
                  <width>600</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(66, 66, 66);</string>
                </property>
               </widget>
               <widget class="SpectogramWidget" name="spectrogram_widget" native="true"/>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Drums</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout">
         <item row="2" column="0">
          <widget class="QLabel" name="drums_label">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>drumss.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QGroupBox" name="groupBox">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>62</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(189, 189, 189);</string>
           </property>
           <property name="title">
            <string/>
           </property>
           <layout class="QGridLayout" name="gridLayout_8">
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QPushButton" name="drums_filter">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>Low Pass</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="drums_slider">
                <property name="minimumSize">
                 <size>
                  <width>400</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="drums_slider_number">
                <property name="minimumSize">
                 <size>
                  <width>150</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;</string>
                </property>
                <property name="text">
                 <string>Volume</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>Xylophone</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="1" column="1" rowspan="3" colspan="2">
          <widget class="QLabel" name="xylo_label">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>xylophone-removebg-preview (1).png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QGroupBox" name="groupBox_2">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>62</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(189, 189, 189);</string>
           </property>
           <property name="title">
            <string/>
           </property>
           <layout class="QGridLayout" name="gridLayout_10">
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QSlider" name="xylo_slider">
                <property name="minimumSize">
                 <size>
                  <width>400</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="xylo_volume_number">
                <property name="minimumSize">
                 <size>
                  <width>150</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;</string>
                </property>
                <property name="text">
                 <string>Volume</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_4">
        <attribute name="title">
         <string>Piano</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QGroupBox" name="groupBox_3">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>62</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(189, 189, 189);</string>
           </property>
           <property name="title">
            <string/>
           </property>
           <layout class="QGridLayout" name="gridLayout_9">
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QPushButton" name="piano_filter">
                <property name="minimumSize">
                 <size>
                  <width>80</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>Low Pass</string>
                </property>
                <property name="checkable">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="piano_slider">
                <property name="minimumSize">
                 <size>
                  <width>400</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="piano_volume_number">
                <property name="minimumSize">
                 <size>
                  <width>150</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;</string>
                </property>
                <property name="text">
                 <string>Volume</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="piano_label">
           <property name="styleSheet">
            <string notr="true">color: rgb(66, 66, 66);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>piano-removebg-preview.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>0</width>
      <height>0</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1117</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_open_2"/>
   </widget>
   <widget class="QMenu" name="menuAbout_Us">
    <property name="title">
     <string>About Us</string>
    </property>
    <addaction name="Reem_Yasser"/>
    <addaction name="Maryam_Moataz"/>
    <addaction name="Hassan_Samy"/>
    <addaction name="Clara_Ashraf"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout_Us"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_open">
   <property name="icon">
    <iconset>
     <normaloff>../../User/.designer/backup/DSP/open file.jpg</normaloff>../../User/.designer/backup/DSP/open file.jpg</iconset>
   </property>
   <property name="text">
    <string>open</string>
   </property>
  </action>
  <action name="action_pause">
   <property name="icon">
    <iconset>
     <normaloff>../../User/.designer/backup/DSP/pause 1.png</normaloff>../../User/.designer/backup/DSP/pause 1.png</iconset>
   </property>
   <property name="text">
    <string>pause</string>
   </property>
  </action>
  <action name="action_play">
   <property name="icon">
    <iconset>
     <normaloff>../../User/.designer/backup/DSP/play.png</normaloff>../../User/.designer/backup/DSP/play.png</iconset>
   </property>
   <property name="text">
    <string>play</string>
   </property>
  </action>
  <action name="action_open_2">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="Reem_Yasser">
   <property name="text">
    <string>Reem Yasser</string>
   </property>
  </action>
  <action name="Maryam_Moataz">
   <property name="text">
    <string>Maryam Moataz</string>
   </property>
  </action>
  <action name="Hassan_Samy">
   <property name="text">
    <string>Hassan Samy</string>
   </property>
  </action>
  <action name="Clara_Ashraf">
   <property name="text">
    <string>Clara Ashraf</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SpectogramWidget</class>
   <extends>QWidget</extends>
   <header location="global">spectrogramwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
